style.css

```diff
@@ -1 +1,217 @@
-/*  Add your styling here */
+/* General body */
+
+body{
+    background-color: rgb(255, 255, 255);
+    font-family: Arial !important;
+}
+
+/* Header / Upper part */
+
+header{
+    width:100%;
+    display: flex;
+    justify-content: space-between;
+    background-color: rgb(255, 212, 229);
+    color: rgb(134, 134, 134);
+    margin-bottom: 20px;
+}
+
+.bakery-icon{
+    width: 100%;
+    display: flex;
+    justify-content: space-around;
+}
+
+.bakery-icon p{
+    font-size: calc(15px + 1vw);
+}
```

Comment:
You have a class for this element: `welcome-msg` so it's best to use that in this case.

Good that you've discovered the `calc()` rule 🙂 however `15px + 1vw` seems like a strange combination here. `vw` is relative to the width of the page, which is probably not relevant for a font-size.